src/classes/STG_InstallScript.cls

```diff
@@ -333,6 +333,31 @@ global without sharing class STG_InstallScript implements InstallHandler {
         }
     }
 
+    /*******************************************************************************************************
+    * @description Ensure that any default Org Rollup Settigns are updated for custom settings 
+    * objects accordingly.  There was a swap of custom setting fields to get the account soft credit settings
+    * pointed to the corresponding setting
+    */
+    public static void updateDefaultRollupSettings() {
+        Customizable_Rollup_Settings__c rollupSettings = UTIL_CustomSettingsFacade.getOrgCustomizableRollupSettings();
+        Boolean isUpdateNeeded = false;
+        Decimal currentAccountSoftCreditBatchSize = UTIL_CustomSettingsFacade.DEFAULT_ROLLUP_BATCH_SIZE;
+
+        if (rollupSettings.Rollups_Account_Soft_Credit_Batch_Size__c != null) {
+            currentAccountSoftCreditBatchSize = rollupSettings.Rollups_Account_Soft_Credit_Batch_Size__c;
+        }
+
+        if (rollupSettings.Rollups_AcctContactSoftCredit_Batch_Size__c == null) {
+            rollupSettings.Rollups_AcctContactSoftCredit_Batch_Size__c = currentAccountSoftCreditBatchSize;
+            isUpdateNeeded = true;
+        }
+        
+        if (isUpdateNeeded) {
+            update rollupSettings;
+        }
+        
+    }
+
```

Comment:
This code will copy the retain the existing Account Contact Soft Credit Batch Size but it will also use that value for the new Account Soft Credit.  I think that is ok.  The alternative could possibly cause future upgrades to overwrite values if the customer accidentally blanks out one of them.